### ISSUE TITLE
Constantly stay on the first logo page

### ISSUE THREAD

**Issue by Miyayx** [vladikoff/chromeos-apk#145]
Dear All

I can load ARChon and APKs successfully.  But when clicked "Launch", it opens an Android page without going on running. Just stay on a page like this 
![Page](https://cloud.githubusercontent.com/assets/1763414/5643109/16b9fad8-968f-11e4-81ed-4325a670f23d.png)

Did anyone encounter the same problem? 

My environment is Fedora 20, Chrome 37+. I have tried both ARChon and ARChon Canary

Hope someone can help me solve the problem. Thank you.

**Comment by ustcltx** [comment]
Can you tell me the name of the app ?

**Comment by Icarua** [comment]
Hello,

I have the same issue in my HP 14 Chromebook (Chrome 39). I can't run any app exept the example 2048 provided by vladikoff. Any idea why this is the case? 

I tried a lot of Apps including Skype and Soundcloud - none working.

**Comment by ustcltx** [comment]
![2015-01-15 14 52 51](https://cloud.githubusercontent.com/assets/6998333/5753555/498c14b8-9cc6-11e4-9928-d6cbcdc78292.png)

Soundcloud works!

You can use the Online Converter:https://huodong.ustc.edu.cn/Crx/

Tutorial & Guide:https://huodong.ustc.edu.cn/Crx/course

**Comment by Icarua** [comment]
Yes - I know that it should work. Somehow I managed to get the Amazon Music App to run properly. I tried a few random Apps, like Cut The Rope and Angry Birds for example, none of them worked unfortunately. I just saw the Logo like the screenshot above shows.

I will try your links this weekend an give you feedback. Thank you so far!

**Comment by Miyayx** [comment]
Unfortunately, even the example 2048 can not work.

Is there anything wrong with my os or chrome?

2015-01-15 17:49 GMT+08:00 Icarua <notifications@github.com>:

> Yes - I know that it should work. Somehow I managed to get the Amazon
> Music App to run properly. I tried a few random Apps, like Cut The Rope and
> Angry Birds for example, none of them worked unfortunately. I just saw the
> Logo like the screenshot above shows.
>
> I will try your links this weekend an give you feedback. Thank you so far!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/vladikoff/chromeos-apk/issues/145#issuecomment-70062056>
> .
>



-- 
Best Wishes

GitHub:github.com/Miyayx

**Comment by yeshourun** [comment]
This may be able to help: https://github.com/ctcos/my-ctcos
I wrote the above guide titled as CTCOS (Contributions to ChromeOS), to help with problems like these. If you follow the steps exactly for your specified problem, you can probably solve the issue.

**Comment by Logiix** [comment]
@yeshourun 

Hi,

I am currently having this problem as well. I followed the instructions of the main page which is the same as the instructions in your guide. I am currently using Chrome version 40.0.2214.115 m (64-bit), and have downloaded v1.2 and 2.1 64-bit and 32-bit to no avail. v2.1 seems to have a bit of response as it shows the dead plugin logo, but with v1.2, the launched app just keep stays on the first logo page.

Are you able to assist in trouble shooting or point me in the direction of a trouble shooting guide?

If you require more information on my system to diagnose the issue, I can provide them to you.

Thank you.

**Comment by arjitkhullar** [comment]
i think you should install play services and then try to start the app.

**Comment by yeshourun** [comment]
@Logiix 

By 'normal runtime', I mean the the Runtime that only ChromeOS users can use (the ARM one), seeing as you're not a ChromeOS user, I recommend that you ignore that. 

You probably shouldn't use anything that is in the Beta stages, so that most likely includes your 40.0.2214.115 m, and everything above that (at the moment). You should download this Stable version: [Intel x86 Chrome 64-bit / Chrome OS 64-bit](http://archon.vf.io/ARChon-v1.2-x86_64.zip) <- Click on the link to download

It also matters how you are getting your APKs. Where are you getting them from, and what are you doing to them to make them Chrome compatible? In case you don't know where to get APKs from, go here: (https://github.com/ctcos/my-ctcos#do-it-yourself). If you have (an) APK/s and don't know how to make it Chrome compatible, use this part of the guide: (https://github.com/ctcos/my-ctcos#what-to-do-with-apks-once-downloaded)

As for the app not starting, even though the app window is open, I can direct you to a part of my guide that has helped people solve this issue. Read the whole Using Twerk: Instructions section: (https://github.com/ctcos/my-ctcos#using-twerk-instructions) You'll most likely have to change a couple lines of code in the `manifest.json` file to make it work. If that doesn't work, than I don't know what will. To find out what lines you should edit in the `manifest.json`, read the whole Twerk section in my guide.  

Please send me some snapshots of your `chrome://extensions` page, and a snapshot of the app not starting. Also, please tell me what app/s you are trying to run, and I can tell you if they are compatible or not.

**Comment by arjitkhullar** [comment]
Instead of using twerk you should go for ARChon Packager.That helped in my case

**Comment by yeshourun** [comment]
@arjitkhullar 

Yes, ARChon Packager is also good, but if you don't have an android phone than it's useless. If you don't know how to use ARChon Packager or where to get it from, read this part of my guide: (https://github.com/ctcos/my-ctcos#using-archon-packager-instructions)

**Comment by arjitkhullar** [comment]
check out this link it may help you.

**Comment by arjitkhullar** [comment]
check out this link https://docs.google.com/spreadsheets/d/1iIbxaftAu_ho5rv9fUlXSLTzwU6MbKOldsWXyrYiyo8/edit#gid=0


it may help you

**Comment by yeshourun** [comment]
That's what I just posted... You can download it here: [Chrome APK Official Apps List](http://www67.zippyshare.com/d/Y8mP7mts/3627/Chrome%20APK%20Official%20Apps%20List.pdf)

**Comment by Logiix** [comment]
@yeshourun @arjitkhullar 

Thanks for the advice.

So I have tried running v1.2 (both 32-bit and 64-bit) ARChon runtime on my chrome before but still did not work.

Also, used the archived APK packages (2048 and pandora) to test whether it runs or not so it should be working as the package was created and have been tested to work. I did try using a package that I made using ARChon packager myself but still doesn't work. The app I tried was a game called Chain Chronicles.

Regarding the app not starting, by that I meant that it gets stuck on the first logo page. Will post screenshots when I am in front of my home computer.

Thanks!

**Comment by Logiix** [comment]
@yeshourun @arjitkhullar 

https://www.dropbox.com/s/o7w8lfzav3a3byu/2048%20after%20the%20app%20reloaded%20twice.jpg?dl=0
https://www.dropbox.com/s/a4c3btg6970i13l/Extensions.jpg?dl=0
https://www.dropbox.com/s/ut87ws4o3lq0yg7/pandora%20after%20the%20app%20reloaded%20twice.jpg?dl=0

Here are the images, I tried to install the Google play service, but could not find it in the extensions or apps from the chrome store on the desktop browser. As such, I found a unpacked APK and loaded that. 

Still no progress =\

**Comment by rkaeferboeck** [comment]
I'm having the same problem.
I get the message: NativeClient: could not find all the resources needed to launch the program but when I click on details there are none being shown... How can I find out which resource is missing?

![screenshot](https://cloud.githubusercontent.com/assets/3436896/6380864/210e644a-bd3d-11e4-81a7-493789107ce8.jpg)

**Comment by rkaeferboeck** [comment]
Problem solved - reason was that I used the Iron-Browser which is based on Chromium but obviously missing something to run ARChon... When I tried it with Chrome it worked immediately!

**Comment by Icarua** [comment]
@arjitkhullar

I already tried the ARChon Packager before - same issue.

I will try changing the linse of code in the manifest.json like Logiix stated above.
But I hope there will be an official support someday.

**Comment by Logiix** [comment]
Hi all,

After testing ARChon runtime 1.2 64-bit on three computers (2 desktop and 1 laptop) I finally got it working on the laptop. Is it possible that ARChon runtime currently doesn't support certain CPUs or perhaps does not support Desktop? This sounds ridiculous but that is the only explanation I can think of seeing as the only common link between the 2 computers that did work are that they are desktops.

**Comment by wokier** [comment]
Hi. I have the same here with Chromium.
I have found that the flags does not offer the "WebGL 2.0 Compute" option

**Comment by illtellyoulater** [comment]
ARChon developers should explicitly state that ARChon is no longer working/supported/maintained!

There's not a single mention of this in hundreds of ARChon official github threads. 
I wasted about 8 full hours trying to make this thing work, being deceived by the their complete lack of communication and the confusion generated by recent posts of other people like me populating these threads in search of answers...

This should be universally considered as the perfect example of how to wrongly manage the discontinuation of a project.
